src/test/msgr/test_msgr.cc

```diff
@@ -37,6 +37,14 @@
 
 typedef boost::mt11213b gen_type;
 
+#include "common/dout.h"
```

Comment:
the change looks fine, but just out of curiosity, why would prefer using logging over cerr in this case?